model/registry/azure.yaml

```diff
@@ -0,0 +1,100 @@
+groups:
+  - id: registry.azure.sdk
+    prefix: az
+    type: attribute_group
+    brief: >
+      This document defines generic attributes for Azure SDK.
+    attributes:
+      - id: service_request_id
+        type: string
+        stability: experimental
+        brief: "The unique reference string that identifies a request"
+        examples:
+          - "00000000-0000-0000-0000-000000000000"
+  - id: registry.azure.logs
+    prefix: azure
+    type: attribute_group
+    brief: >
+      This document defines generic attributes for Azure Logs.
+    attributes:
+      - id: category
```

Comment:
If we are modeling azure log as an event, the category is effectively an event name. 

If there are cases it's needed as an attribute, it'd suggest `az.log.category` to make it more precise.

Reply:
This is fixed now we've addd the `az.log` prefix.